### DISCUSSION TITLE
Track actions in combat

### DISCUSSION THREAD

**Discussion by Cikoni-git** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#128] [Ideas]
It would be great if you could toggle and track how many actions each actor has used in combat.
For example in the encounter interface have two buttons that you can toggle as slow or fast action and these buttons are reset in the next round.

**Comment by aMediocreDad** [comment]
Hey there. This is something I want to do in a combat tracker overhaul. However until then, there is a module that adds this for you already: https://foundryvtt.com/packages/alien-actions

**Reply by Cikoni-git** [reply]
Hey aMediocreDad, Thanks so much! I wasn't aware of this module.
